client/src/pages/Profile.js

```diff
@@ -23,31 +25,34 @@ const Image = styled(ProfileImage)`
 const InputContainer = styled.div`
   display: flex;
   flex-flow: column nowrap;
+  justify-content: space-around;
   align-items: center;
-  justify-content: space-evenly;
   height: 50%;
-  width: 80vw;
+  width: 100vw;
```

Comment:
don't use `vw` here, this element is at a lower level it should get it's size from one of its ancestor elements.